sdk/core/src/test/java/com/klaviyo/coresdk/networking/requests/IdentifyApiRequestTest.kt

```diff
@@ -0,0 +1,146 @@
+package com.klaviyo.coresdk.networking.requests
+
+import android.util.Base64
+import com.klaviyo.coresdk.BaseTest
+import com.klaviyo.coresdk.model.Profile
+import io.mockk.every
+import io.mockk.mockkStatic
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNull
+import org.junit.Assert.fail
+import org.junit.Test
+
+internal class IdentifyApiRequestTest : BaseTest() {
+    private val expectedUrlPath = "api/identify"
+    private val expectedMethod = RequestMethod.GET
+    private val encodedQueryString = "Expected json string, base64 encoded"
+    private val expectedQueryData = mapOf("data" to encodedQueryString)
+    private var profile = Profile()
+
+    override fun setup() {
+        super.setup()
+
+        // Start all tests with an empty profile and base64 mock for encoding it
+        profile.setAnonymousId(ANON_ID)
+        withMockBase64("{\"properties\":{\"\$anonymous\":\"$ANON_ID\"},\"token\":\"$API_KEY\"}")
+    }
+
+    private fun withMockBase64(expectedString: String) {
```

Comment:
I don't think you need this if you use the post request.

Reply:
Sorry meant to comment on the actual request no the test.

Reply:
I do have a ticket to update to the V3 API, is that what you mean?

Reply:
Oh christ you mean there's a v2 `identify` endpoint that just takes POST with a body?? I wasted like half a day figuring out how to write those stupid base64 mocked tests.

Reply:
Sorry, frustration directed at myself for not looking at the docs closer, not you.